elide-core/src/main/java/com/yahoo/elide/security/executors/ActivePermissionExecutor.java

```diff
@@ -116,6 +116,16 @@ public ActivePermissionExecutor(boolean verbose, final com.yahoo.elide.core.Requ
         };
 
         Function<Expression, ExpressionResult> expressionExecutor = (expression) -> {
+            // for newly created object in PatchRequest limit to User checks
```

Comment:
I would say something like:

We only run User checks during patch extension newly created object because there is a good chance
that inline checks may fail (relationships are not yet fixed up).  If the user checks return deferred, we will defer all of the inline checks to commit phase.  It is safe to defer inline checks in this case because the
object was newly created and all values excluded computed attributes were provided by the client.